astropy/coordinates/builtin_frames/cirs_observed_transforms.py

```diff
@@ -97,30 +106,22 @@ def altaz_to_cirs(altaz_coo, cirs_frame):
     cirs_ra, cirs_dec = erfa.atoiq('A', az, zen, astrom)*u.radian
 
     if isinstance(altaz_coo.data, UnitSphericalRepresentation):
```

Comment:
same as above: unlikely but possible, so I will switch them all to the same idiom